app/helpers/application_helper.rb

```diff
@@ -1,6 +1,37 @@
 module ApplicationHelper
+  include Pagy::Frontend
   # NOTE: move to specific datetime place (forget name)
   def pretty_datetime(timestamp)
     timestamp.strftime('%B %d, %Y at %l:%M %p')
   end
+
+  def pagy_nav(pagy, pagy_id: nil, link_extra: '', **vars)
```

Comment:
need to overwrite the `pagy_nav` helper so that it uses tailwind/daisyUI classes.
Please ignore the fact that I'm not using the native DaisyUI pagination, it just didn't work for some reason and it made less sense to try to fight with daisy than just use native tailwind.